router/core/graphql_prehandler.go

```diff
@@ -429,7 +448,7 @@ func (h *PreHandler) handleOperation(req *http.Request, variablesParser *astjson
 	}
 
 	// Compute the operation sha256 hash as soon as possible for observability reasons
-	if h.computeOperationSha256 {
+	if h.shouldComputeHash(operationKit) {
```

Comment:
Since we have multiple hashes in the code, can we be more specific here e.g. sha256?